horace_core/sqw/@sqw/sqw.m

```diff
@@ -77,28 +75,26 @@
         w = sigvar_set(win, sigvar_obj);
         sz = sigvar_size(w);
         %------------------------------------------------------------------
+        wout=signal(w,name)  % Set the intensity of an sqw object to the
+        %                    % values for the named argument
```

Comment:
Is it better to call this set_signal? SIgnal sounds like it is returning the signal values.

Reply:
The method name should remain `signal` for backwards compatibility. Also, it _does_ return the signal, but as a new `sqw` object rather than as a vector. The docstring here should probably say `% Returns a new sqw object with its intensity set to the values of the named argument` - e.g. we are not setting the intensity of `w`, but rather the intensity of a new object `wout`.

Reply:
see answer above